crates/rust-analyzer/src/config.rs

```diff
@@ -296,6 +301,8 @@ pub struct Config {
     detached_files: Vec<AbsPathBuf>,
     pub discovered_projects: Option<Vec<ProjectManifest>>,
     pub root_path: AbsPathBuf,
+    postfix_snippets: Vec<PostfixSnippet>,
+    snippets: Vec<Snippet>,
 }
```

Comment:
These two get their own extra vecs so we don't have to rederive them from the raw string inputs on every completion as that may slow things down.